bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.9.0
+    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
```

Comment:
```
    - uses: jpkrohling/setup-operator-sdk@v1.1.0
      with:
        operator-sdk-version: v1.9.0
```

The operator-sdk in CI Is generating the version as `v1.8.0+git`. So changed it back but open to make changes to `operator-sdk` github action to generate the version as `v1.9.0`

Reply:
Sounds reasonable to me. cc @owais